src/open_inwoner/openklant/services.py

```diff
@@ -846,6 +846,9 @@ class OpenKlant2Service(KlantenService):
 
     def __init__(self, config: OpenKlant2Config | None = None):
         self.config = config or OpenKlant2Config.get_solo()
+        if not self.config.service:
```

Comment:
```suggestion
        if not getattr(self.config, "service", None):
```
since `service` is nullable?

Reply:
I think the semantics of `get_solo()` means it's always `get_or_create` under the hood, no?

Reply:
You're right, didn't realize this applies to all fields